Geodata_CityGML/Geodata_CityGML_shacl.ttl

```diff
@@ -0,0 +1,57 @@
+@prefix Geodata_CityGML: <https://github.com/GAIA-X4PLC-AAD/map-and-scenario-data/tools/ontologie_creator/ontologies/Geodata_CityGML_ontology/> .
+@prefix sh: <http://www.w3.org/ns/shacl#> .
+@prefix skos: <http://www.w3.org/2004/02/skos/core#> .
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
+
+Geodata_CityGML:Asset_Shape a sh:NodeShape ;
```

Comment:
change prefix 
Asset_Shape should be GeodataCityGmlShape or GeoDataCityGmlShape